tools/run_tests/xds_k8s_test_driver/run.sh

```diff
@@ -68,4 +68,8 @@ fi
 
 cd "${XDS_K8S_DRIVER_DIR}"
 export PYTHONPATH="${XDS_K8S_DRIVER_DIR}"
-exec python "$@" --flagfile="${XDS_K8S_CONFIG}"
+# Split path to python file from the rest of the args.
+readonly PY_FILE="$1"
+shift
+# Append args after --flagfile, so they take higher priority.
+exec python "${PY_FILE}" --flagfile="${XDS_K8S_CONFIG}" "$@"
```

Comment:
Potentially, you could use `${@:2}` to skip the first item.

Reply:
TIL. Though, using `shift` is a common pattern too